internal/tools/inspect/app.go

```diff
@@ -55,7 +55,8 @@ func newApp(ctx context.Context, l *slog.Logger, j job) (*app, error) {
 	data := struct{ Version string }{
 		Version: "v" + a.AppVer.String(),
 	}
-	if err = j.Renderer.Render(a.tmpDir, data); err != nil {
+	err = j.Renderer.Render(a.tmpDir, data)
+	if err != nil {
```

Comment:
Can you explain me this change?

Reply:
I disagree with `noinlineerr` authors opinion. Let's please not enable it  :wink: 

Resolving this conversation.

Thanks for all your answers and contribution!